.github/workflows/ci.yml

```diff
@@ -82,37 +80,50 @@ jobs:
         phpize
         ./configure
         make
-        make test
-
-
-  #   linux-arm:
-  #     name: ARM (${{ matrix.arch }})
-  #     runs-on: ubuntu-latest
-  #     strategy:
-  #       matrix:
-  #         arch: [armv6, armv7, arm64]
-  #     steps:
-  #     - name: Build ${{ env.PACKAGE_NAME }}
-  #       run: |
-  #         python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
-  #         chmod a+x builder
-  #         ./builder build -p ${{ env.PACKAGE_NAME }} --target=linux-${{ matrix.arch }} --spec=downstream
-
-  # windows-vc16:
-  #   runs-on: windows-latest
-  #   strategy:
-  #     matrix:
-  #       arch: [x64]
-  #   steps:
-  #   - uses: ilammy/msvc-dev-cmd@v1
-  #     with:
-  #       arch: ${{ matrix.arch }}
-  #       uwp: false
-  #       spectre: true
-  #   - name: Build ${{ env.PACKAGE_NAME }} + consumers
-  #     run: |
-  #       python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
-  #       python builder.pyz build -p ${{ env.PACKAGE_NAME }} --spec=downstream
+        ./dev-scripts/run_tests
+
+    # linux-arm:
+    #   name: ARM (${{ matrix.arch }})
+    #   runs-on: ubuntu-latest
+    #   strategy:
+    #     matrix:
+    #       arch: [armv6, armv7, arm64]
+    #   steps:
+    #   - name: Build ${{ env.PACKAGE_NAME }}
+    #     run: |
+    #       python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
+    #       chmod a+x builder
+    #       ./builder build -p ${{ env.PACKAGE_NAME }} --target=linux-${{ matrix.arch }} --spec=downstream
+
+  windows-vc16:
+    runs-on: windows-2019
+    defaults:
+      run:
+        shell: cmd # use CMD instead of powershell to catch error from bat script
+    strategy:
+      matrix:
+        arch: [x64]
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - id: setup-php-sdk
+      uses: cmb69/setup-php-sdk@v0.6
+      with:
+        version: '8.0'
+        arch: x64
+        ts: ts
+        deps: openssl
```

Comment:
Confused: What is this action? I could not find anything about it using a quick Google search (https://github.com/marketplace?type=actions&query=Setup+PHP+). Can you please add some documentation for why we need a different action for Windows CI? It appears that shivammathur/setup-php@v2 also supports Windows.

Reply:
TBH, I didn't dig deep. I followed the PR initialed from Mike, https://github.com/awslabs/aws-crt-php/pull/77/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR114.
And I believe Mike found it from https://github.com/Imagick/imagick/blob/master/.github/workflows/windows.yml

I was really just want to have something working...